src/Uno.UI/LayoutProvider.Android.cs

```diff
@@ -50,6 +54,13 @@ internal void Start(View view)
 				_adjustResizeLayoutProvider.Start(view);
 			}
 		}
+
+		internal void StartListenInsets()
+		{
+			_adjustNothingLayoutProvider.StartListenInsets();
```

Comment:
Given that you don't even pass a insets listener to `_adjustNothingLayoutProvider`, do you need to listen for insets on it?

Reply:
The NavigationBarRect is based on this Rect so I probably need to listen changes on it in order to update the NavigationBarRect, right ?

Reply:
`NavigationBarRect` doesnt seems to use the insets from `_adjustNothingLayoutProvider`, at least not directly. The question would be whether `return insets.ConsumeSystemWindowInsets();` on the adjust-nothing popup is necessary. _I think_, you can skip the listen part entirely on `_adjustNothingLayoutProvider`.